submissions/zenzelya/dom-api-task/src/js/index.js

```diff
@@ -0,0 +1,49 @@
+// another js file (example)
```

Comment:
Why do you upload two files `index.js` in the same directory? Please, get rid of this